qiskit_experiments/framework/base_experiment.py

```diff
@@ -267,9 +267,7 @@ def run(
         run_opts = run_opts.__dict__
 
         # Generate and transpile circuits
-        transpile_opts = copy.copy(experiment.transpile_options.__dict__)
-        transpile_opts["initial_layout"] = list(experiment.physical_qubits)
-        circuits = transpile(experiment.circuits(), experiment.backend, **transpile_opts)
+        circuits = experiment.transpiled_circuits()
```

Comment:
perhaps this can be renamed to `physical_circuits` to explicitly say this is transpiled, i.e. logical vs physical in conventional compiler terminology.

Reply:
I see what you mean, but I still prefer `transpiled` because I'm not sure how much the reader is familiar with compiler terminology, but I expect the reader to know what "transpile" means.

Reply:
How about caching the transpiled circuits? I often visualize transpiled circuits before running an experiment. In that case, `transpiled_circuits()` is called twice. With caching, we can speed up the second run.
```
# in __init___()
self._transpiled_circuits = None  

# in run()
if self._transpiled_circuits is None:
      self._transpiled_circuits = experiment.transpiled_circuits()
circuits = self._transpiled_circuits
```

Reply:
Make sense. Does lru cache do the same?